### PR TITLE
Issue210 use cases

### PR DESCRIPTION
**Pull request by lcoram**

### CODE REVIEW

**Comment by mortenwh** [comment]
I can't find the source of this error:
```
WARNING: error while running plantuml

b'ERROR\n2\nSyntax error: LAYOUT_LEFT_RIGHT\nSome diagram description contains errors\n'
```
Do you know where it's coming from?

**Review by mortenwh — CHANGES_REQUESTED** [review]
Then we need to talk to Anita :)

**Comment by mortenwh** [comment]
Todo:

- [ ] talk to Anita

- [ ] Answer questions in the note under *Actors*

- [ ] take out old use cases

**Comment by mortenwh** [comment]
We also need to work a bit more on the normal flow - and consider adding an alternative flow in case of existing challenges..

File: source/use_case_climate_projection.rst [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+   execution of the Use Case under normal, expected conditions. This dialog sequence will ultimately
+   lead to accomplishing the goal stated in the Use Case name and description.
+
+* The researcher investigates the data
+
+  * downloads the climate predictions for temperature for the whole time range.
+  * downloads the climate prediction medians as an average over the period 2071-2100.
+
+* The journalist choses parameters on a website to se what the extreme temperatures will be in 2041-2070.
+* The state agency updates their maps with expected changes, to reflect future changes in water flow.
```
Comment (mortenwh) [inline source/use_case_climate_projection.rst:111]: we should change the decision here - how can a decision be relevant to changing temperature?

Comment (lcoram) [reply source/use_case_climate_projection.rst:111]: Temperature effects planting conditions ... maybe something about farms and tree planting for wood?

Comment (mortenwh) [reply source/use_case_climate_projection.rst:111]: I tried to link it to permafrost - I'll push a new commit soon

**Comment by havardf** [comment]
In the climate projection use-case the mention of OGC WMS and netcdf comes out without any deeper explanation. I am thinking that that this is either:
- A constraint imposed on the project from some higher up decision.
- Or an implementation choice, and hence should not be in the use-case description. 

In the latter case, it should be clear from the use-case description that the data must be available in a form that is easy to find and consume for the given consumer, and so when the solution is implemented they would have to figure out how that should be handled.

Also, there are other requirements for the implementation besides the use-case description right? To avoid that a use-case desc. does not transform into a big up-front requirements spec.

**Comment by lcoram** [comment]
I would potentially like to "wash" the use case of all the specifics like file formats and particular websites (although maybe that is a requirement of climate 2100?). But I think Morten wanted us to understand why they were using that format, and where they get the data today and also how it is distributed today.

**Review by mortenwh — APPROVED** [review]

**Pull request merged**